### ISSUE TITLE
Update all tutorials to reflect async/await refactors to IPFS API

### ISSUE THREAD

**Issue by terichadbourne** [ProtoSchool/protoschool.github.io#347]
@alanshaw called out that the async/await changes being made to js-ipfs in https://github.com/ipfs/js-ipfs/pull/2683 will make our lesson content outdated. (Nothing on the site will actually break until we update IPFS versions.) I'll schedule some time for us to chat about the scope of the changes and who's best equipped to make updates.

**Comment by alanshaw** [comment]
js-ipfs 0.41.0-rc.0 is out now if this was a blocker at all....

**Comment by zebateira** [comment]
Thank you @alanshaw 🙏 

I poked around your [migration guide](https://gist.github.com/alanshaw/04b2ddc35a6fff25c040c011ac6acf26), and from what I could tell, there are two breaking changes that we need to deal with: `ipfs.add` and `ipfs.cat`

- `ipfs.cat`
  - https://proto.school/#/regular-files-api/04
  - https://proto.school/#/regular-files-api/07
- `ipfs.add`
  - https://proto.school/#/regular-files-api/03
  - https://proto.school/#/regular-files-api/04
  - https://proto.school/#/regular-files-api/05
  - https://proto.school/#/regular-files-api/06
  - https://proto.school/#/regular-files-api/07

So basically, almost the entire Regular Files API tutorial.

@alanshaw @terichadbourne would appreciate your opinion on this: in `.add` and `.cat` we now need to pipe/concat all the data of a file.
To update the tutorials code exercises, I'm thinking of two options:

1. We could simply mention why we need to do this and then simply add the `it-concat` and `it-pipe` modules to make the code solutions easier.
2. Another option would be to already include the `for await ...` loops in the code solution with its code block empty ready for the users to fill in the solution code.

What do you think?
